subprojects/gradle/build-metrics/src/test/kotlin/com/avito/android/plugin/build_metrics/BuildMetricTrackerTest.kt

```diff
@@ -0,0 +1,84 @@
+package com.avito.android.plugin.build_metrics
+
+import com.avito.android.plugin.build_metrics.internal.BuildStatus
+import com.avito.android.sentry.StubEnvironmentInfo
+import com.avito.android.stats.SeriesName
+import com.avito.android.stats.StubStatsdSender
+import com.avito.android.stats.TimeMetric
+import com.avito.utils.gradle.Environment
+import com.google.common.truth.Truth.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class BuildMetricTrackerTest {
+
+    private lateinit var statsd: StubStatsdSender
+
+    @BeforeEach
+    fun setup() {
+        statsd = StubStatsdSender()
+    }
+
+    @Test
+    fun `build - local environment`() {
+        val tracker = metricTracker(
+            environment = Environment.Local,
+            node = "user"
+        )
+        tracker.track(TimeMetric(SeriesName.create("metric"), 1))
+
+        val metrics = statsd.getSentMetrics()
+        assertThat(metrics).hasSize(1)
+        assertThat(metrics.first().name.toString()).isEqualTo("local.user.build.metric")
+    }
+
+    @Test
+    fun `build - CI environment`() {
+        val tracker = metricTracker(
+            environment = Environment.CI,
+            node = "agent-3c23034b"
```

Comment:
why node value is missing in assertion? was it intended?

Reply:
Yes, it's intended behavior. A node in CI is ephemeral because of a container. It's no use in metrics.

Added to test name.